### PR TITLE
Issue #5489: Fixed indentation in config.xml

### PR DESCRIPTION
**Pull request by varunu28**
Updated config.xml for #5489

### CODE REVIEW

File: src/xdocs/config.xml [hunk 1]
```diff
@@ -242,9 +242,7 @@
 
       <source>
 &lt;module name=&quot;JavadocMethod&quot;&gt;
-  &lt;property name=&quot;severity&quot;
-               value=&quot;${checkstyle.javadoc.severity}&quot;
-               default=&quot;error&quot;/&gt;
+  &lt;property name=&quot;severity&quot; value=&quot;${checkstyle.javadoc.severity}&quot; default=&quot;error&quot;/&gt;
```
Comment (romani) [inline src/xdocs/config.xml]: keep ALL multiline - it is easier to read.
just change indentation.

Comment (rnveach) [inline src/xdocs/config.xml]: Line must remain under 100 characters.
We don't have this rule integrated by CI yet.
Please ensure all lines you change remain under 100.

Comment (rnveach) [reply src/xdocs/config.xml]: Jinx.

**Comment by romani** [comment]
> Should all indentation like this be part of the issue?

@rnveach , ideally - yes.
@varunu28 , please keep 2 spaces indentation. It is ok to changes from 4 spaces to 2 spaces in separate PRs, even for the same file, to keep changes easy to review.

**Comment by varunu28** [comment]
@romani @rnveach So everything a module block should have an indentation of 2?

**Comment by romani** [comment]
You can do this in this PR or next PR, it was just noticed, so it up you. What ever is easier for you.

**Comment by varunu28** [comment]
@romani I will do that in this PR itself. Let us try to fix the as many indentation issues possible in one go

File: src/xdocs/config.xml (hunk 1 above)
Comment (varunu28) [reply src/xdocs/config.xml]: done

Comment (varunu28) [reply src/xdocs/config.xml]: done

**Comment by varunu28** [comment]
@romani @rnveach I have made the suggested changes. I have also changed the indentation from 4 to 2. Please let me know what changes need to be done. If it looks good then I can go ahead and squash the commits

File: src/xdocs/config.xml [hunk 2]
```diff
@@ -869,8 +871,8 @@
 
       <source>
 &lt;!DOCTYPE module PUBLIC
-    &quot;-//Puppy Crawl//DTD Check Configuration 1.3//EN&quot;
-    &quot;http://checkstyle.sourceforge.net/dtds/configuration_1_3.dtd&quot;&gt;
+&quot;-//Puppy Crawl//DTD Check Configuration 1.3//EN&quot;
+&quot;http://checkstyle.sourceforge.net/dtds/configuration_1_3.dtd&quot;&gt;
```
Comment (romani) [inline src/xdocs/config.xml]: Should there be indentation +2 ?

Comment (varunu28) [reply src/xdocs/config.xml]: I did it based on the existing version (line 913)

```
      <source>
&lt;!DOCTYPE checkstyle-packages PUBLIC
&quot;-//Puppy Crawl//DTD Package Names 1.1//EN&quot;
&quot;http://checkstyle.sourceforge.net/dtds/packages_1_1.dtd&quot;&gt;
      </source>
```

Does it needs to be changed to +2 indentation after the DOCTYPE? If yes then I would do the changes at both the places?

Comment (varunu28) [reply src/xdocs/config.xml]: done

**Comment by romani** [comment]
Please squash all in one for final review

**Comment by rnveach** [comment]
@varunu28 I find it easier to always squash commits when I am working a PR. It is only in rare cases we have more than 1 commit in PR.
Since you are doing 1 file per PR, you should only ever have 1 commit.

**Comment by varunu28** [comment]
@rnveach I am working on squashing the commits. I want to go with the squash once I get a green signal from you and @romani. Will add a squashed commit in next few hours. Thank you

**Comment by varunu28** [comment]
@romani @rnveach Squashed the commits

**Comment by romani** [comment]
Diff looks good, as I come to keyboard , I will check look of html page.

**Comment by rnveach** [comment]
> I will check look of html page.

Feel free to merge if it is all ok.

**Comment by varunu28** [comment]
@romani Please merge this

**Comment by romani** [comment]
looks good, please continue in such a format.

**Pull request merged**